src/sourceItemWrapper.js

```diff
@@ -489,20 +489,92 @@ class SourceItemWrapper extends ItemWrapper {
         // offer to automatically link to zotero items
     }
 
-    getFromBibTeX() {
-        Services.prompt.alert(
-            window,
-            Wikicite.getString('wikicite.global.unsupported'),
-            Wikicite.getString('wikicite.bibtex.import-citations.unsupported')
+    // get BibTeX from clipboard
+    // - also supports other formats supported by Zotero
+    // - also supports multiple items
+    async getFromBibTeX() {
+        var str = Zotero.Utilities.Internal.getClipboard("text/unicode");
+        var translation = new Zotero.Translate.Import();
+        translation.setString(str);
+
+        const progress = new Progress(
+            'loading',
+            'Adding citations...'
         );
+        
+        try {
+            var translators = await translation.getTranslators();
+
+            if (translators.length > 0){
+                // set libraryID to false so we don't save this item in the Zotero library
+                var newItems = await translation.translate({libraryID: false});
+        
+                let citations = []
+                if (newItems.length > 0){
+                    newItems.forEach((item) => {
+                        let newItem = new Zotero.Item(item.itemType);
+                        newItem.fromJSON(item);
+            
+                        let citation = new Citation({item: newItem, ocis: []}, this);
+                        citation.target.item = newItem;
```

Comment:
I think this one is redundant.

Reply:
When removing these two lines and directly assigning jsonItem to the item field in new Citation, I receive an error that notes is not a valid field - in ItemWrapper.fromJSON. I tried deleting `notes` `seeAlso` and `attachments` from the jsonItem as in wikidata, but this still wasn't successful. I guess I'm just doing something wrong?

Reply:
I meant to anchor my comment to line 519, but now I see it's ambiguous what line I was referring to. I think the redundant line is `citation.target.item = newItem`. The citation's target item should be defined at instantiation on line 518. Does this make sense?

I'm entering a meeting now. I'll reply to your other comments in a moment. Thanks!!